great_expectations/agent/models.py

```diff
@@ -0,0 +1,21 @@
+from typing import Literal, Union
+from typing_extensions import Annotated
```

Comment:
Until we drop `python 3.7` we are bound to an earlier version of `typing_extentions` due to an `airflow` compatibility problem.
So you might not be able to use `Annotated` at runtime.

But we are dropping `python 3.7` within the next few weeks so It's probably not worth making too much effort with a workaround if it leads to worse code.

Reply:
looks like 3.7 will get dropped just in time for this to merge 👌